src/widgets/visit/new-visit-component.tsx

```diff
@@ -0,0 +1,347 @@
+          dayjs(new Date(visit.visitData.startDatetime)).format("YYYY-MM-DD")
+        );
+        setVisitStartTime(
+          dayjs(new Date(visit.visitData.startDatetime)).format("HH:mm")
+        );
+        visit.visitData.stopDatetime &&
+          setVisitEndDate(
+            dayjs(new Date(visit.visitData.stopDatetime)).format("YYYY-MM-DD")
+          );
+        visit.visitData.stopDatetime &&
+          setVisitEndTime(
+            dayjs(new Date(visit.visitData.stopDatetime)).format("HH:mm")
+          );
+
+        setVisitTypeUuid(visit.visitData.visitType.uuid);
+      }
+    });
+
+    return () => sub && sub.unsubscribe();
+  }, [props.viewMode]);
+
+  const newVisitView = () => {
+    return (
+      <SummaryCard name="Starting New Visit" styles={{ margin: 0 }}>
+        <div className={styles.newVisitContainer}>
+          <div
+            className={`${styles.newVisitInputContainer} ${styles.flexColumn}`}
+          >
+            <label htmlFor="visitType">Visit Type</label>
+            <VisitTypeSelect
+              onVisitTypeChanged={visitType =>
+                onVisitTypeChanged(visitType.uuid)
+              }
+              id={"visitType"}
+              visitTypeUuid={visitTypeUuid}
+            />
+          </div>
+          <div
+            className={`${styles.newVisitInputContainer} ${styles.flexColumn}`}
+          >
+            <label htmlFor="startDate">Start Date/Time</label>
+            <div
+              className={styles.flexRow}
+              style={{ display: "flex", padding: "0rem 0.25rem" }}
+            >
+              <input
+                type="date"
+                name="startDate"
+                id="startDate"
+                data-testid="date-select"
+                defaultValue={visitStartDate}
+                onChange={onStartDateChanged}
+                style={{ flex: 1, marginRight: "0.625rem" }}
+              />
+              <input
+                type="time"
+                name="startTime"
+                id="startTime"
+                data-testid="time-select"
+                defaultValue={visitStartTime}
+                onChange={onStartTimeChanged}
+                style={{ flex: 1, marginLeft: "0.625rem" }}
+              />
+            </div>
+          </div>
+          <div
+            className={`${styles.newVisitInputContainer} ${styles.flexColumn}`}
+          >
+            <label htmlFor="location">Location</label>
+            <LocationSelect
+              currentLocationUuid={locationUuid}
+              onLocationChanged={location => onLocationChanged(location.uuid)}
+              id={"location"}
+            />
+          </div>
+          <div
+            className={styles.newVisitButtonContainer}
+            style={{ flexDirection: "row" }}
+          >
+            <button
+              className={`omrs-btn omrs-outlined-neutral`}
+              onClick={() => props.onCanceled()}
+            >
+              Cancel
```

Comment:
All these boil also down to localization. I think we should try to put more emphasis on this. Changing it later can be tedious.